### PR TITLE
Dec. 2020 news

### PR DESCRIPTION
**Pull request by aeslaughter**
@idaholab/moose-team Please add your Nov. contributions. If you also have stuff for Dec. you can add it in the Jan. 2021 now as well.

(refs #11447)

### CODE REVIEW

File: modules/doc/content/newsletter/2020_12.md [hunk 1]
```diff
@@ -2,4 +2,16 @@
 
 ## Migration to Github Discussions
 
-The MOOSE development team decided to move all new questions to a [Github discussion forum](https://github.com/idaholab/moose/discussions).
+The MOOSE development team decided to move all new questions to a [Github discussion
```
Comment (cticenhour) [inline modules/doc/content/newsletter/2020_12.md:5]:
```suggestion
The MOOSE development team has decided to move all new questions to a [Github discussion
```

File: modules/doc/content/newsletter/2020_12.md [hunk 2]
```diff
@@ -2,4 +2,16 @@
...
+forum](https://github.com/idaholab/moose/discussions).
```
Comment (cticenhour) [inline modules/doc/content/newsletter/2020_12.md:6]: This link doesn't seem to want to render in the website preview (https://mooseframework.inl.gov/docs/PRs/16457/site/newsletter/2020_12.html), so I think you might want this all on a single line.

File: modules/doc/content/newsletter/2021_01.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# MOOSE Newsletter (January 2021)
+
+# Improved INL HPC Remote Connectivity
+
+The [hpc_remote.md] instructions have been updated for a better user experience in accessing web-based INL HPC resources. The previous instructions involved using an additional browser with a proxy to access said resources (hpcgitlab, moosebuild, hpcweb, etc). The new instructions utilize a proxy configuration that is much easier to setup for the end-user and do not require an additional browser. For more information, see [hpc_remote.md#socks-proxy].
```
Comment (cticenhour) [inline modules/doc/content/newsletter/2021_01.md:5]:
```suggestion
The [hpc_remote.md] instructions have been updated for a better user experience in accessing web-based INL HPC resources. The previous instructions involved using an additional browser with a proxy to access said resources (hpcgitlab, moosebuild, hpcweb, etc). The new instructions utilize a proxy configuration that is much easier to setup for the end-user and does not require an additional browser. For more information, see [hpc_remote.md#socks-proxy].
```

File: modules/doc/content/newsletter/2021_01.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+The module traces rays throughout the finite element mesh. Notable features include:
+
+- Ray contribution to residuals and Jacobians along ray segments (example: line sources)
+- Tracing on meshes with adaptivity
+- Supports ray interaction with internal and external boundaries
+- Supports storage and manipulation of data unique to each ray
+- Supports ray interaction with field variables
+- Supports parallel execution
+
+The pictured example is an anisotropic point source that emits in a cone of directions. See [flashlight_source.md] for more information.
```
Comment (cticenhour) [inline modules/doc/content/newsletter/2021_01.md:22]: "Cone of directions" sounds a bit awkward to me, without considering the pictured example. Which directions, or what direction?

**Review by cticenhour — CHANGES_REQUESTED** [review]

**Comment by fdkong** [comment]
I will add something soon

**Comment by lindsayad** [comment]
Ok my stuff is added

File: modules/doc/content/newsletter/2021_01.md (hunk 3 above)
Comment (loganharbour) [reply modules/doc/content/newsletter/2021_01.md:5]:
```
The new instructions ... do not
```

vs

```
The new instructions ... does not
```

?

Comment (cticenhour) [reply modules/doc/content/newsletter/2021_01.md:5]: eh, I was thinking "A proxy configuration that does not..."

Comment (loganharbour) [reply modules/doc/content/newsletter/2021_01.md:5]: Yeah - I actually had this same struggle when I wrote this. Maybe I should just try to rewrite the sentence

Comment (cticenhour) [reply modules/doc/content/newsletter/2021_01.md:5]: It's a clause within the larger sentence, so I think my change is grammatically correct.

File: modules/doc/content/newsletter/2021_01.md (hunk 4 above)
Comment (loganharbour) [reply modules/doc/content/newsletter/2021_01.md:22]:
```
The pictured example is an anisotropic point source that is nonzero only in a cone
```

How about that?

Comment (cticenhour) [reply modules/doc/content/newsletter/2021_01.md:22]: I think that sounds better. The fact that this cone is made up of multiple rays with multiple directions is explained in the linked doco, so keeping this a tiny bit more general is probably best for a newsletter.

File: modules/doc/content/newsletter/2021_01.md (hunk 3 above)
Comment (cticenhour) [reply modules/doc/content/newsletter/2021_01.md:5]: Maybe something like:
```suggestion
The [hpc_remote.md] instructions have been updated for a better user experience in accessing web-based INL HPC resources. The previous instructions involved using an additional browser with a proxy to access said resources (hpcgitlab, moosebuild, hpcweb, etc). The new proxy configuration file method is much easier to setup for the end-user and does not require an additional browser. For more information, see [hpc_remote.md#socks-proxy].
```

Comment (loganharbour) [reply modules/doc/content/newsletter/2021_01.md:5]: You win 👍🏼. I like.

**Comment by WilkAndy** [comment]
Perhaps someone could add this:

## Paper describing the PorousFlow module published

A paper describing the core physics in the PorousFlow module has been published.  It also outlines a number of PorousFlow models that have been built and published over the past few years, including groundwater models, geothermal simulations, multi-phase modelling assessments and mineralization problems.

https://joss.theoj.org/papers/10.21105/joss.02176

## Geochemistry module added

MOOSE now contains a physics module that can solve geochemical models. The capabilities include:

- equilibrium aqueous systems
- redox disequilibrium
- sorption and surface complexation
- kinetics
- all these combined with fluid and heat transport

The geochemistry module is designed to interface easily with the porous-flow module so that complicated reactive transport scenarios can be studied.  The [documentation](geochemistry/index.md) contains many fully-explained examples to help you get started with geochemistry.

**Comment by loganharbour** [comment]
Sure thing @WilkAndy. I'm about to make some changes. Will add.

**Comment by WilkAndy** [comment]
> Sure thing @WilkAndy. I'm about to make some changes. Will add.

Thanks @loganharbour

File: modules/doc/content/newsletter/2021_01.md (hunk 4 above)
Comment (loganharbour) [reply modules/doc/content/newsletter/2021_01.md:22]: I opted to just remove it. Pretty picture is enough as it is detailed on the main module page.

File: modules/doc/content/newsletter/2020_12.md [hunk 5]
```diff
@@ -2,4 +2,47 @@
...
+- Redox disequilibrium
+- Sorption and surface complexation
+- Kinetics
+- All of the above combined with fluid and heat transport
+
+The geochemistry module is designed to interface easily with the porous flow module so that complicated reactive transport scenarios can be studied. The [documentation](geochemistry/index.md) contains many detailed examples to help you get started with geochemistry.
+
+## Heap Profiling
+
+We added support to do heap profiling using `gperftools`. The capability is useful for tracking memory spikes in both serial and parallel simulations. Instructions for using this capability are at [here](https://mooseframework.inl.gov/application_development/profiling.html).
```
Comment (cticenhour) [inline modules/doc/content/newsletter/2020_12.md:43]:
```suggestion
Support has been added to do heap profiling using `gperftools`. This is useful for tracking memory spikes in both serial and parallel simulations. Instructions for using this capability can be found[here](https://mooseframework.inl.gov/application_development/profiling.html).
```

File: modules/doc/content/newsletter/2020_12.md [hunk 6]
```diff
@@ -2,4 +2,47 @@
...
+
+## Fixed an Memory Issue in Assembly::init
+
+There were several fully coupled flag matrices used in `Assembly`, which generated memory spikes for the simulation that has
+many moose `nonlinear variables`. We fixed the issue by creating the diagonal flag matrices if there are not coupling elements between moose `nonlinear variables`, which is often true when using `PJFNK`. Preconditioner might not need a fully coupled matrix for certain problems.
```
Comment (cticenhour) [inline modules/doc/content/newsletter/2020_12.md:48]:
```suggestion
There were several fully coupled flag matrices used in `Assembly` which generated memory spikes for simulations that have
many MOOSE `nonlinear variables`. We fixed the issue by creating diagonal flag matrices if there are not coupling elements between MOOSE `nonlinear variables`, which is often true when using `PJFNK`. Preconditioners might also not need a fully coupled matrix for certain problems. 
```

File: modules/doc/content/newsletter/2020_12.md [hunk 7]
```diff
@@ -2,4 +2,47 @@
...
+
+## MultiApp UserObject Transfer
+
+`MultiAppUserObjectTransfer` can now be restricted to a block or a boundary using `block` or
+`boundary` parameter.  This helps to combine multiple transfers into a single variable which
+can also simplify the overall simulation setup of kernels, BCs, etc.
+
+## Scaling Support for Global AD Indexing
+
+We have added both automatic and manual scaling support for MOOSE's global AD indexing
```
Comment (cticenhour) [inline modules/doc/content/newsletter/2020_12.md:21]:
```suggestion
Support was added for both automatic and manual scaling in MOOSE's global AD indexing
```

**Review by cticenhour — CHANGES_REQUESTED** [review]
A few more comments

**Comment by cticenhour** [comment]
@aeslaughter There are two suggestions I made on your contribution that are outstanding. If you address those, I think this is ready to go, unless you think any more content will be added.

**Review by cticenhour — APPROVED** [review]

**Pull request merged**